src/prelude.rs

```diff
@@ -607,6 +607,32 @@ pub fn parse_paths(
             None,
         ));
     }
+
+    // NOTE.2022-10-26 - Heroic flatpak installation detection
+    //
+    // flatpak wiki on filesystems
+    // (https://github.com/flatpak/flatpak/wiki/Filesystem) as well as
+    // https://docs.flatpak.org do not seem to mention an option to relocate
+    // per-app data directories.  These are by default located in
+    // $HOME/.var/app/$FLATPAK_ID, so we cat detect a flatpak installed heroic
+    // by looking at the `root_interpreted` and check for
+    // ".var/app/com.heroicgameslauncher.hgl/config/heroic"
+    if root.store == Store::Heroic
+        && get_os() == Os::Linux
+        && root_interpreted.ends_with(".var/app/com.heroicgameslauncher.hgl/config/heroic")
```

Comment:
I added `assert_eq!("/foo/bar".to_string(), interpret("/foo/bar/", &None));` to `path::can_interpret_linux_style_paths` and it succeeds.  Since `root_interpreted` is the result of `intepret`, is it safe to assume that a possible trailing / was removed?

Reply:
Yeah, I'm fine with that 👍